src/modules/profile/profile.controller.ts

```diff
@@ -0,0 +1,102 @@
+import { APIResponse } from "#/src/lib/types/misc";
+import { toResponse } from "#/src/lib/utils";
+import { statusConst } from "#/src/lib/utils/status";
+import { Request as ExpressRequest } from "express"; // Explicitly import from Express
+import {
+  Controller,
+  Delete,
+  FormField,
+  Get,
+  Patch,
+  Request,
+  Route,
+  Security,
+  Tags,
+  UploadedFile,
+} from "tsoa";
+import { getReqUser } from "../auth/auth.helpers";
+
+import fileService from "../file/file.service";
+import { sanitizeUser } from "../user/user.helpers";
+import userService from "../user/user.service";
+import { SanitizedUser } from "../user/user.types";
+import { profileService } from "./profile.services";
+@Route("profile")
+@Tags("Profile")
+@Security("jwt")
+export class ProfileController extends Controller {
+  @Get("/")
+  public async fetch(
+    @Request() req: ExpressRequest
+  ): Promise<APIResponse<SanitizedUser>> {
+    const reqUser = getReqUser(req);
+
+    const user = await userService.fetch(reqUser.id);
+    if (!user) {
+      this.setStatus(statusConst.notFound.code);
+      return toResponse({ error: statusConst.notFound.message });
+    }
+
+    return toResponse({ data: sanitizeUser(user) });
+  }
+  @Patch("/")
```

Comment:
Add a one line gap after each controller, function, etc.
```suggestion
  @Patch("/")

```